src/jit/codegenxarch.cpp

```diff
@@ -8808,4 +8183,116 @@ void CodeGen::genAmd64EmitterUnitTests()
 
 #endif // _TARGET_AMD64_
 
+const CodeGen::GenConditionDesc& CodeGen::GetConditionDesc(GenCondition condition)
+{
+    // clang-format off
+    static constexpr GenConditionDesc map[]
```

Comment:
It's possible to flip compares (via `GenCondition::Reverse`) but that's not related to this table. This is based on the existing codegen approach to floating point compares - unfortunately some cannot be implemented using a single SETcc/Jcc. For example `bool b = float1 == float2;` is
```asm
vucomiss    xmm0,xmm1
setnp       al
jp          00007FFA1F3804DD
sete        al
```
Expanding these in lower is costly (we need to split the basic block) and unlikely to buy as anything so such expansion is still deferred to codegen.

I need to add comments to this table to show how entries map to `GenCondition` codes.